packages/pubsub/src/PubSub.ts

```diff
@@ -151,6 +153,12 @@ export class PubSubClass {
 		topics: string[] | string,
 		options?: ProvidertOptions
 	): Observable<any> {
+		if (isNode) {
+			throw new Error(
+				'Subscriptions are not supported in Node to prevent memory leaks.'
```

Comment:
I'm leaning towards the following @sammartinez :

```suggestion
				'Subscriptions are not supported in Node.'
```